### ISSUE TITLE
Creation of dynamic property warnings on PHP 8

### ISSUE THREAD

**Issue by stgoos** [spivurno/gfur-activate-template#7]
When testing with error reporting enabled I received some "creation of dynamic property" warnings. I'm using PHP 8.x.

To solve this add the 2 public declarations to the class in `activate.php`.

Also, `__file__` should be `__FILE__`.

```
class GWActivateTemplate {

    public $template_folder; 
    public $result;

    function __construct( $args = array() ) {

        /**
         * @var $template_folder
         */
        extract( wp_parse_args( $args, array(
            'template_folder' => basename( dirname( __FILE__ ) )
		) ) );
```

**Comment by spivurno** [comment]
@stgoos Thanks for the feedback! If you submit a PR, I'll be happy to get this merged. 🙌

**Comment by stgoos** [comment]
I created a PR for it, hopefully I did everything right as it's my first PR ever :)

**Comment by spivurno** [comment]
Fixed by #8.
